spec/shopping_list_spec.rb

```diff
@@ -6,8 +6,15 @@
 
   let(:app) { Application }
 
-  it 'returns 200'
-  it 'gives an empty shopping list'
+  it 'returns 200' do
+    get '/shopping_list'
+    expect(last_response.status).to eq 200
+  end
+
+  it 'gives an empty shopping list' do
+    get '/shopping_list'
+    expect(ListItem.all).to be_empty
```

Comment:
we agreed that, even though this reaches into the model layer, this is the best test for this behavior right now.

we could consider rewriting once some details about the HTML rendering are done